.changes/next-release/feature-2152796a-9dd7-4889-9d00-3f56da0b2002.json

```diff
@@ -0,0 +1,4 @@
+{
+  "type" : "feature",
+  "description" : "Adding Pause and Resume Auto Code Scans Buttons to status bar menu"
+}
```

Comment:
Please remove this, we will add a single description when the feature branch is merged to `main` branch.